docs/config.md

```diff
@@ -334,8 +404,9 @@ we'd want to use 1199120 bytes. As of writing, [7 hash
 functions](https://github.com/ipfs/go-ipfs-blockstore/blob/547442836ade055cc114b562a3cc193d4e57c884/caching.go#L22)
 are used, so the constant `k` is 7 in the formula.
 
+Default: `0` (disabled)
```

Comment:
Aside: do we have any interest in "reclaiming" the 0 value to mean `default` now and using another indicator (e.g. -1, false, etc.) to indicate disabled? Doing this in docs now may make it easier for us to update in the future given that we'll know the user's preference.

On the other hand, resolving https://github.com/ipfs/go-ipfs-config/issues/53 and is going to be a bigger endeavor that will likely involve users needing to update their config files, so small things like this might be irrelevant.

Reply:
I think we need to move away from the whole "zero value" hack in general. In this case, `0` _should_ mean disabled.